src/index.html

```diff
@@ -4,10 +4,58 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__photo"
+        src="images/imac.jpeg"
+        alt="MAC"
+       >
+      <div class="card__description">
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```

Comment:
Put an empty line in such cases
```suggestion
       >
       
      <div class="card__description">
        <div class="card__name">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```